Pavel_Karabanovich/02.GIT.Local/02.GIT.Local.md

```diff
@@ -0,0 +1,162 @@
+
+### Homework 02.Git.Local by Karabanovich_Pavel
+
+#### Git command history
+```bash
+git init
+nano config
+nano .gitignore
+git status
+date > test1.txt
+cat test1.txt
+touch test.log
+git add --all
+git commit -m "commit1"
+git log --oneline
+git log
+git branch
+date > test2.txt
+cat test2.txt
+git status
+git add --all
+git commit -m "commit2"
+git checkout -b dev 
+touch test3.txt
+date > test3.txt
+git add --all
+git commit -m "commit3"
+touch test4.txt
+date > test4.txt
+git add --all 
+git commit -m "commit4"
+git log --oneline
+git branch 
+git checkout -b features/do_one 
+date >> test1.txt 
+git add test1.txt 
+git commit -m "commit5"
+git checkout master
+git branch
+date > test2.txt 
+git commit -m "commit6"
+```
+#### Git command history for PLAY1 "Release phase"
+```bash
+git checkout master
+git merge dev
+git merge features/do_one
+git log --oneline
+```
+#### List of branch and commit for PLAY1
+
+streamx1@ubuntu-222:~/dz2.git$ git branch
+
```

Comment:
Please continue use git markdown here as well.
The report have to be fully formated

Reply:
I`ve done it. Please check. Thanks!